lib/game.rb

```diff
@@ -0,0 +1,176 @@
+module PlanetExpress
+  class Game
+
+    def initialize
+      @bender = PlanetExpress::Bender.new(self)
+      @fry = PlanetExpress::Fry.new(self)
+      @hermes = PlanetExpress::Hermes.new(self)
+      @leela = PlanetExpress::Leela.new(self)
+      @zoidberg = PlanetExpress::Zoidberg.new(self)
+      @stable = true
+    end
+
+    def play_game
+      loop do
+        @bender = PlanetExpress::Bender.new(self)
+        @fry = PlanetExpress::Fry.new(self)
+        @hermes = PlanetExpress::Hermes.new(self)
+        @leela = PlanetExpress::Leela.new(self)
+        @zoidberg = PlanetExpress::Zoidberg.new(self)
+        @stable = true
+        do_play_game
+        puts "Do you want to play again? (y/n)"
+        break unless gets.chomp == "y"
+      end
+    end
+
+    def do_play_game
```

Comment:
It seems like this is the method that should handle all of the game flow, so I don't quite understand why there are `check` and `prompt` method calls scattered around other locations.